src/Symfony/Component/Notifier/Bridge/Lox24/.gitattributes

```diff
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
```

Comment:
please add a new line to make the red pictogram disappear

Same for the other

Thanks

Reply:
Should i add new line at the end of file?

Reply:
Yes, you can configure your IDE to always do this

Reply:
done